### PR TITLE
Fix ajax in 5.5 and lower

### PR DESCRIPTION
**Pull request by jkrocil**
{{pytest: -k test_login -v }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit e064552109033099ea4f406ff0dc5989059a8fa0:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 42034e3df1e3f566ccec57c22ef8bbe56d01be08:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/js.py [hunk 1]
```diff
@@ -20,8 +20,8 @@
     spinner: (!isHidden(document.getElementById("spinner_div")))
         && isHidden(document.getElementById("lightbox_div")),
     document: document.readyState,
-    autofocus: checkMiqQE('autofocus'),
-    miqQE: checkAllMiqQE()
+    autofocus: typeof(checkMiqQE) === undefined ? checkMiqQE('autofocus') : 0,
+    miqQE: typeof(checkAllMiqQE) === undefined ? checkAllMiqQE() : 0
```
Comment (mfalesni) [inline cfme/js.py]:
```javascript
typeof checkMiqQE === "undefined"
```

Since typeof is an operator and it returns strings.

Comment (jkrocil) [inline cfme/js.py]: http://stackoverflow.com/questions/3390396/how-to-check-for-undefined-in-javascript

Comment (jkrocil) [inline cfme/js.py]: Long story short - It's javascript, nobody's gonna redefine undefined, it doesn't matter.

Comment (mfalesni) [inline cfme/js.py]:
```bash
$ js
```

```javascript
typeof asdf === undefined
false
```

:smirk:

Comment (mfalesni) [inline cfme/js.py]: === is like python's `is`.

**Comment by dajoRH** [comment]
Lint report for commit b585a8b025045edfe5b8eae6a91b6cfb9615b22c:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 79cc36d1cb7f3e9e5097c1c77848ff721e8cf8ce:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**